modules/images-configuration-registry-mirror-configuring.adoc

```diff
@@ -21,6 +26,13 @@ endif::openshift-rosa,openshift-dedicated[]
 .Procedure
 
 . Configure mirrored repositories, by either:
+ifdef::winc[]
```

Comment:
I might move this warning to the beginning of the procedure or even before it so that it doesn't break apart the step, but your call.